app/src/main/java/is/xyz/mpv/MPVActivity.kt

```diff
@@ -126,6 +131,19 @@ class MPVActivity : Activity(), MPVLib.EventObserver, TouchGesturesObserver {
         controls.nextBtn.setOnLongClickListener { pickPlaylist(); true }
     }
 
+    private fun dpadCenterTriggerAction(btnId: Int) {
+        when (btnId) {
```

Comment:
This should definitely call `onClick()` (or whatever the name is) of the selected view directly.
That'd make the when statement unnecessary and also be "portable" for future changes.

Reply:
Way better indeed.
`controls_button_group.getChildAt(btnSelected)?.performClick()`